lib/demo_mode/cli.rb

```diff
@@ -48,12 +48,9 @@ def prompt_persona # rubocop:disable Metrics/AbcSize
                 puts "👉 #{feature}"
               end
 
-              named_tags = SemanticLogger.named_tags
-
               variant = variant_for(persona, persona_name)
 
               CLI::UI::Spinner.spin("generating account...") do |spinner|
-                SemanticLogger.push_named_tags(named_tags)
```

Comment:
Worth adding a CLI-related callback so that consuming apps can add this back if they want? I imagine there was a reason we added it originally.

(or is there a different callback clients could use?)

Reply:
`CLI::UI::Spinner` runs the the block in a new thread. I'm guessing that we're copying the logger's tags to the new thread.

Rather than deleting this code, I decided to just guard against the possibility that `SemanticLogger` isn't defined. Properly fixing this problem would involve either:
1. Introducing some sort of hook/callback, which would complicate this library.
2. Choosing a spinner library that doesn't run our code in a new thread.